index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="wrapper">
+    <header class="first">
+      <nav class="menu">
+        <ul>
+          <li><a href="#" class="logo">Logo</a></li>
+          <li><a href="#"><img src="./images/union.svg" alt="Hamburger logo"></a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="second">
+      <h1>I’m Yersel<br>
+        Glad to see you!
+      </h1>
+      <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h2>LET'S CONNECT</h2>
+      <div class="icons">
+        <ul>
+          <li><a href="#"><img src="./images/twitter.svg" alt="Twitter logo"></a></li>
+          <li><a href="#"><img src="./images/Linkedin.svg" alt="Linkedin logo"></a></li>
+          <li><a href="#"><img src="./images/m.svg" alt="m logo"></a></li>
+          <li><a href="#"><img src="./images/github.svg" alt="github logo"></a></li>
+          <li><a href="#"><img src="./images/hands.svg" alt="hands logo"></a></li>
```

Comment:
- [ ] OPTIONAL: Great job implementing the `<a>` for each link.  I suggest you kindly change the "#"s on each `href=` for the actual links to your social media.  This way, people interested in you will be able to connect. 🥇